kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkConnector.java

```diff
@@ -102,7 +102,9 @@ private SchemaManager getSchemaManager(BigQuery bigQuery) {
     }
     SchemaRetriever schemaRetriever = config.getSchemaRetriever();
     SchemaConverter<com.google.cloud.bigquery.Schema> schemaConverter = config.getSchemaConverter();
-    return new SchemaManager(schemaRetriever, schemaConverter, bigQuery);
+    boolean includeKafkaKey = config.getBoolean(config.INCLUDE_KAFKA_KEY_CONFIG);
+    boolean includeKafkaData = config.getBoolean(config.INCLUDE_KAFKA_DATA_CONFIG);
```

Comment:
Just curious, when would we like to set `includeKafkaData` to `false`?

Reply:
I'm not sure, `includeKafkaData` is an existing config and it's default to be `false`..

Reply:
Yea-- it's just an option. Some folks might not care, and would prefer to have their Avro match their BQ table schemas exactly (no extra kafkaData field). Just an option.

Reply:
Got it!